plugin/src/ServeSession.lua

```diff
@@ -207,6 +247,162 @@ function ServeSession:__onActiveScriptChanged(activeScript)
 	self.__apiContext:open(scriptId)
 end
 
+function ServeSession:__replaceInstances(idList)
+	if #idList == 0 then
+		return true, PatchSet.newEmpty()
+	end
+	-- It would be annoying if selection went away, so we try to preserve it.
```

Comment:
🐐

Reply:
(Spoiler: we don't) https://github.com/boatbomber/rojo/blob/master/plugin/src/Reconciler/applyPatch.lua#L118